frontend-new/src/feedback/feedbackForm/components/customRating/CustomRating.tsx

```diff
@@ -93,26 +89,27 @@ const CustomRating: React.FC<CustomRatingProps> = ({
                 color: theme.palette.text.secondary,
               }}
             >
-              <Typography variant="caption" data-testid={DATA_TEST_ID.CUSTOM_RATING_LOW_LABEL}>
+              <Typography variant="caption" data-testid={DATA_TEST_ID.CUSTOM_RATING_LOW_LABEL}
+                          sx={{ fontSize: theme.fixedSpacing(theme.tabiyaSpacing.sm) }}>
```

Comment:
I think sm might be a little too small here... It looks nice, but hardly readable

Reply:
maybe just on mobile